types/gapi.people/gapi.people-tests.ts

```diff
@@ -16,9 +16,9 @@
   function checkAuth() {
     gapi.auth.authorize(
       {
-        'client_id': CLIENT_ID,
-        'scope': SCOPES.join(' '),
-        'immediate': true
+        client_id: CLIENT_ID,
+        scope: SCOPES.join(' '),
+        immediate: true
```

Comment:
not related to the goal of this PR but removing quotes to make it more consistent with other gapi packages like  [this one](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/gapi.auth2/gapi.auth2-tests.ts) (gapi uses both tho 🤷 ). also quote on object props in this case doesn't seem necessary